andela/ah-the-answer-frontend
# 164829343 - Highlighting and commenting on a section.

#### What does this PR do?
Implements the highlighting and commenting on sections of the article
![image](https://user-images.githubusercontent.com/12595401/58690506-5d94af80-8392-11e9-80f1-26a4286e865c.png)

#### Description of the task to be completed
- Select a section of the test
- Comment and highlight a section

#### How should this be manually tested
- Clone the repo and navigate to the root directory using a terminal
- Install all the dependencies by typing `npm install`
- Start the server by typing `npm start`
- Navigate to the main page using a browser at `http://localhost:3000/`
- Open an article.
- Highlight a section, a popup should open and comment on this.
